llvm/include/llvm/Transforms/Utils/LoopUtils.h

```diff
@@ -414,6 +417,9 @@ Value *createTargetReduction(IRBuilderBase &B, const RecurrenceDescriptor &Desc,
 Value *createOrderedReduction(IRBuilderBase &B,
                               const RecurrenceDescriptor &Desc, Value *Src,
                               Value *Start);
+Value *createOrderedReduction(VectorBuilder &VB,
```

Comment:
nit: might be good to highlight that this version is used to create vector-predication intrinsics?

Reply:
518051b849bdf0148ff1ec3b499da667873eec0a